### DISCUSSION TITLE
route

### DISCUSSION THREAD

**Discussion by KeyurShah96** [postalserver/postal#1654] [Help with using Postal]
I want to create a new route. so how can I create and where I can get a document for the route so that I can find out what is the use and other details

**Comment by willpower232** [comment]
What are you trying to do with Postal?

**Reply by KeyurShah96** [reply]
I want to create a route in postal

**Reply by willpower232** [reply]
You should be able to go here in the UI, you'll need to add an endpoint (i.e. a destination) and then you can create the route.

![image](https://user-images.githubusercontent.com/1619102/138655269-96216a92-aa9c-4b0a-b07a-0e4d473b4e19.png)
